container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/engine_functions/create_engine.go

```diff
@@ -27,6 +30,11 @@ const (
 	maxWaitForEngineAvailabilityRetries         = 10
 	timeBetweenWaitForEngineAvailabilityRetries = 1 * time.Second
 	logsStorageDirpath                          = "/var/log/kurtosis/"
+	removeLogsWaitHours                         = 6
```

Comment:
nit: you can directly do `6 * time.Hour` and avoid having `*Hours` in your variable name (which will be a duration)